docs/docs-content/cluster-templates/cluster-templates.md

```diff
@@ -0,0 +1,138 @@
+and [macros](../clusters/cluster-management/macros.md), allowing you to deploy, manage, and scale a synchronized fleet
+of clusters with minimal effort while configuring environment-specific values where needed.
+
+Cluster templates can be created at both the tenant and project scope. Cluster templates do not embed cluster profile
+and policy configurations but reference them as objects, allowing you to edit and replace them as needed.
+
+Cluster templates can be managed using [Terraform](../automation/terraform/terraform.md),
+[Crossplane](../automation/crossplane/crossplane.md), the [Palette Go SDK](../automation/palette-sdk/palette-sdk.md),
+and [APIs](/api/introduction).
+
+![Diagram of cluster template architecture](/cluster-templates_diagram.webp)
+
+## Cluster Profile Behavior
+
+Cluster templates link to [cluster profiles](../profiles/cluster-profiles/cluster-profiles.md), which remain the
+foundation of deploying clusters in Palette. Each cluster template must link to one
+[full](../profiles/cluster-profiles/create-cluster-profiles/create-full-profile.md) or
+[infrastructure profile](../profiles/cluster-profiles/create-cluster-profiles/create-infrastructure-profile.md) and one
+[maintenance policy](./create-cluster-template-policies/maintenance-policy.md). You can reference as many
+[add-on profiles](../profiles/cluster-profiles/create-cluster-profiles/create-addon-profile/create-addon-profile.md) as
+desired. Only one policy of each type can be linked per cluster template (for example, one maintenance policy per
+template).
+
+Once a cluster profile is linked to a cluster template, that version of the cluster profile becomes immutable.
+
+![Linked cluster profile version locked](/cluster-templates_locked-cluster-profile.webp)
+
+To make changes to a linked cluster profile, you must
+[create a new version](../profiles/cluster-profiles/modify-cluster-profiles/version-cluster-profile.md) of the cluster
+profile and update the version referenced in the cluster template. This ensures that all clusters using the cluster
+template have identical software stacks, preventing configuration drift that can naturally occur with inline, on-the-fly
+updates, or by using varied cluster profile versions. Each cluster can be attached to only one cluster template at a
+time, further guarding against drift.
+
+A cluster profile version cannot be linked to a cluster template if that profile version is already being used by a
+cluster that is not attached to a cluster template.
+
+### Cluster Profile Variables
+
+Since cluster profile versions are immutable once linked to a cluster template,
+[cluster profile variables](../profiles/cluster-profiles/create-cluster-profiles/define-profile-variables/define-profile-variables.md)
+remain the recommended way to configure environment-specific values per cluster, such as IPs, resource limits, and more.
```

Comment:
"remain" feels a bit negative no? Feels like "we (spectro) broke something, but here's a workaround"

Reply:
```suggestion
are the best way to configure environment-specific values per cluster, such as IPs, resource limits, and more.
```